src/website/app/demos/Menu/examples/menu/menu.js

```diff
@@ -22,11 +22,11 @@ import DemoLayout from '../../components/DemoLayout';
 
 export default {
   id: 'menu',
-  title: 'Menu',
+  title: 'Example',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description:
-    'Menus are composed of [MenuDivider](../menu-divider), [MenuGroup](../menu-group), and [MenuItem](../menu-item).',
+  description: `Menus are composed of [MenuDivider](../menu-divider), [MenuGroup](../menu-group), and [MenuItem](../menu-item).
+Menus display a list of actions available depending on context.`,
```

Comment:
"Menus display a list of actions available depending on context." ➡️  "Menus display a list of actions or navigation options." ?

Reply:
updated